src/renderer/utils/__tests__/jsonPath.test.tsx

```diff
@@ -0,0 +1,22 @@
+import { parseJsonPath } from "../jsonPath";
+
+describe("parseJsonPath", () => {
+  test("should convert \. to use indexed notation", () => {
+    const res = parseJsonPath(".metadata.labels.kubesphere\\.io/alias-name");
+
+    expect(res).toBe(".metadata['labels']['kubesphere.io/alias-name']");
```

Comment:
It seems weird that `labels` would be also converted to indexed notation, I guess it doesn't matter much but the test name implies something else.

Reply:
Fixed it now, so that only those keys which have illegal characters are converted